.github/workflows/rust.yml

```diff
@@ -18,6 +17,8 @@ env:
 
 jobs:
   mac:
+    # Run CI only on master in upstream
+    if: ${{ !(github.repository == 'servo/servo' && github.event_name == 'push') || github.ref_name == 'master' }}
```

Comment:
I'm guessing `servo/servo` here should be `servo/mozjs`?

Reply:
Yes